src/orange/g4org/Volume.hh

```diff
@@ -0,0 +1,66 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2024 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file orange/g4org/Volume.hh
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <memory>
+#include <vector>
+
+#include "orange/OrangeTypes.hh"
+#include "orange/transform/VariantTransform.hh"
+
+class G4LogicalVolume;
+
+namespace celeritas
+{
+namespace orangeinp
+{
+class ObjectInterface;
+}
+
+namespace g4org
+{
+//---------------------------------------------------------------------------//
+struct LogicalVolume;
+
+//---------------------------------------------------------------------------//
+/*!
+ * An unconstructed Object with a transform.
+ */
+struct PhysicalVolume
+{
+    std::string name;
+    size_type copy_number{};
+    VariantTransform transform;
+    std::shared_ptr<LogicalVolume const> lv;
+};
+
+//---------------------------------------------------------------------------//
+/*!
+ * A reusable Object that can be turned into a UnitProto or a Material.
```

Comment:
Here and throughout I think it would be helpful to be more specific with terminology since there is so much (often unavoidable) clash. In other words: "an ORANGE UnitProto or Geant4 Material".